extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -59,13 +59,13 @@ var NodeChatSocketWrapper = $.createClass(Observable, {
 
 			socket.on('message', this.proxy(this.onMsgReceived, this));
 			socket.on('connect', this.proxy(function () {
-				this.onConnect(socket);
+				this.onConnect(socket, ['xhr-polling']);
 			}, this));
 			socket.on('reconnecting', connectionFail);
 		});
 	},
 
-	onConnect: function (socket) {
+	onConnect: function (socket, transport) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "transport". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)